src/appshell/qml/Preferences/internal/CommonAudioApiConfiguration.qml

```diff
@@ -81,22 +75,20 @@ Item {
             }
         }
 
-        // ComboBoxWithTitle {
-        //     id: bufferSize
-
-        //     title: qsTrc("appshell", "Buffer size:")
-        //     columnWidth: root.columnWidth
+        ComboBoxWithTitle {
+            title: qsTrc("appshell/preferences", "Recording channels")
+            columnWidth: root.columnWidth
 
-        //     currentIndex: indexOfValue(apiModel.bufferSize)
-        //     model: apiModel.bufferSizeList
+            currentIndex: indexOfValue(apiModel.currentInputChannels)
+            model: apiModel.inputChannelsList
 
-        //     navigation.name: "BufferSizeBox"
-        //     navigation.panel: root.navigation
-        //     navigation.row: root.navigationOrderStart + 1
+            navigation.name: "RecordingChannelsBox"
+            navigation.panel: root.navigation
+            navigation.row: root.navigationOrderStart + 1
```

Comment:
Could you please fix navigation for RecordingDevice (start + 1) and update this one (start + 2)?
(and please check the navigation for others controls)